### PR TITLE
Docs did not match build output path

### PR DESCRIPTION
**Pull request by hahla**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -57,8 +57,7 @@ You must have Go > 1.6 installed.
     make build
 
 And to run hoverfly:
-
-    ./cmd/hoverfly/hoverfly
+    $GOPATH/bin/hoverflyfb
```
Comment (mogronalol) [inline readme.md]: Thanks for spotting this.  Your changes are correct, barring it should be `$GOPATH/bin/hoverflyb` instead of `$GOPATH/bin/hoverflyfb`.

Comment (tjcunliffe) [inline readme.md]: Hi - docs are currently being rewritten. They should be in the next release. I'll be updating/improving the build instructions, so might leave this PR where it is for now... thanks though!

**Comment by hahla** [comment]
@mogronalol sorry about that, fixed

**Comment by mogronalol** [comment]
No problem, I'll merge it in but we are likely replace the whole file with the new documentation shortly.

**Pull request merged**